Examples/image-transform-demo.lisp

```diff
@@ -0,0 +1,101 @@
+(defpackage :image-transform-demo
+  (:use :cl)
+  (:export :image-transform-demo))
+
+(in-package :image-transform-demo)
+
+(clim:define-application-frame image-transform-demo ()
+  ((image :initarg :image
+          :accessor image-transform-demo/image))
+  (:panes (image-demo :application
+                      :display-function 'display-image-demo)
+          (rot-slider :slider
+                      :value 0
+                      :show-value t
+                      :orientation :horizontal
+                      :min-value 0
+                      :max-value (* pi 2)
+                      :value-changed-callback #'slider-updated-callback)
+          (x-slider :slider
+                    :value 0
+                    :show-value t
+                    :orientation :horizontal
+                    :min-value 0
+                    :max-value 200
+                    :value-changed-callback #'slider-updated-callback)
+          (y-slider :slider
+                    :value 0
+                    :show-value t
+                    :orientation :horizontal
+                    :min-value 0
+                    :max-value 200
+                    :value-changed-callback #'slider-updated-callback)
+          (scale-slider :slider
+                        :value 1
+                        :show-value t
+                        :orientation :horizontal
+                        :min-value 0.1
+                        :max-value 5
+                        :value-changed-callback #'slider-updated-callback)
+          (x-skew-slider :slider
+                         :value 0
+                         :show-value t
+                         :orientation :horizontal
+                         :min-value -1
+                         :max-value 1
+                         :value-changed-callback #'slider-updated-callback)
+          (y-skew-slider :slider
+                         :value 0
+                         :show-value t
+                         :orientation :horizontal
+                         :min-value -1
+                         :max-value 1
+                         :value-changed-callback #'slider-updated-callback))
+  (:layouts (default (clim:vertically (:width 1000 :height 900)
```

Comment:
Please avoid using explicit pixel values for `:height` and `:width`. (my laptops display height is 768). You can try my modified layout:
```common-lisp
(:layouts (default (clim:vertically ()
		       (14/20 (clim:labelling (:label "Test Image"
					       :align-x :center
					       :label-alignment :top)
				image-demo))
		       (clim:horizontally ()
			 (clim:vertically ()
			   (1/20 (clim:labelling (:label "Rotate")
				   rot-slider))
			   (1/20 (clim:labelling (:label "Translate X")
				   x-slider)))
			 (clim:vertically ()
			   (1/20 (clim:labelling (:label "Translate Y")
				   y-slider))
			   (1/20 (clim:labelling (:label "Scale")
				   scale-slider)))
			 (clim:vertically ()
			   (1/20 (clim:labelling (:label "Skew X")
				   x-skew-slider))
			   (1/20 (clim:labelling (:label "Skew Y")
				   y-skew-slider)))))))
```